code.gs

```diff
@@ -1690,6 +1690,37 @@ var i18n = {
     'It looks like you are using an outdated version of this script': 'ดูเหมือนว่าคุณกำลังใช้เวอร์ชั่นเก่าสำหรับสคริปท์นี้',
     'You can find the latest one here': 'คุณสามารถหาเวอร์ชั่นใหม่ได้ที่นี่',
   },
+  'tr': {
+    'Age': 'Yaş',
+    'Years': 'Yıl',
+    'Events': 'Etkinlikler',
+    'Birthdays today': 'Bugünkü doğum günleri',
+    'Birthdays tomorrow': 'Yarınki doğum günleri',
+    'Birthdays in {0} days': '{0} gün içindeki doğum günleri',
+    'Anniversaries today': 'Bugünkü yıldönümleri',
+    'Anniversaries tomorrow': 'Yarınki yıldönümleri',
+    'Anniversaries in {0} days': '{0} gün içindeki yıldönümleri',
+    'Custom events today': 'Bugünkü özel etkinlikler',
+    'Custom events tomorrow': 'Yarınki özel etkinlikler',
+    'Custom events in {0} days': '{0} gün içindeki özel etkinlikler',
+    'Hey! Don\'t forget these events': 'Hey! Bu etkinlikleri unutma!',
+    'version': 'sürüm',
+    'dd-MM-yyyy': 'gg-AA-yyyy',
```

Comment:
This particular string should not be "translated" in the strict sense of the word, but instead it should be used to alter the format of the date.

For example in Italy we usually write the date as DAY/MONTH/YEAR, so this string should is "dd/mm/yyyy" in the italian translation. In the USA instead the usually write it as MONTH/DAY/YEAR, so the translation for this string should read "mm/dd/yyyy".

Please reorder the parameters of this string to adapt to whatever standard is used in your country.

You are not the first translator to change this string, so I'll add a comment next to it exaplaining this.

Reply:
ok, fixing it